src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -176,6 +176,13 @@ $sidenav-icon-padding-size: 5px;
   }
 }
 
+:deep(.nav__background) {
+  background-color: var(--color-fill);
```

Comment:
https://github.com/apple/swift-docc-render/pull/792/commits/677e5be61b87011d9ed18ad982fbdccfff7cfdfa

Reply:
We should provide a solid color for the DocumentationNav but we cannot use the `hasSolidBackground` prop because we would lose the blur when the nav is sticking.

I refactored it to be cleaner: https://github.com/apple/swift-docc-render/pull/792/commits/32692c649eff5f0c9a1bab7fc42d517a44c59e49